types.go

```diff
@@ -913,12 +996,37 @@ func makeDecl(ti typeInfo) string {
 		default:
 			panic("invalid size of FLNNTYPE")
 		}
+	case typeDecimal, typeDecimalN:
```

Comment:
(I'm personally okay with this getting merged without decimal / numeric support. There isn't a one-sized all option yet for decimal in go :/ .)